[Bug]: config file load expected

### Current Behavior

Absolute config file path, load not works as expected.

```console
./milvus-backup --config $(pwd)/backup.yaml create 


config:/root/tmp/downloads/backup.yaml
panic: cannot access config file: /home/runner/work/milvus-backup/milvus-backup/core/paramtable/../../configs//root/tmp/downloads/backup.yaml

goroutine 1 [running]:
github.com/zilliztech/milvus-backup/core/paramtable.(*BaseTable).LoadYaml(0xc0001cda00, {0x7ffdb112e391, 0x1f})
        /home/runner/work/milvus-backup/milvus-backup/core/paramtable/base_table.go:179 +0x41b
github.com/zilliztech/milvus-backup/core/paramtable.(*BaseTable).loadFromYaml(...)
        /home/runner/work/milvus-backup/milvus-backup/core/paramtable/base_table.go:127
github.com/zilliztech/milvus-backup/core/paramtable.(*BaseTable).Init(0xc0001cda00)
        /home/runner/work/milvus-backup/milvus-backup/core/paramtable/base_table.go:85 +0x174
github.com/zilliztech/milvus-backup/core/paramtable.(*BaseTable).GlobalInitWithYaml.func1()
        /home/runner/work/milvus-backup/milvus-backup/core/paramtable/base_table.go:77 +0x4a
sync.(*Once).doSlow(0x27?, 0xc0005c3518?)
        /opt/hostedtoolcache/go/1.18.10/x64/src/sync/once.go:68 +0xc2
sync.(*Once).Do(...)
        /opt/hostedtoolcache/go/1.18.10/x64/src/sync/once.go:59
github.com/zilliztech/milvus-backup/core/paramtable.(*BaseTable).GlobalInitWithYaml(0x10c8ca0?, {0x7ffdb112e391?, 0xc0005c3858?})
        /home/runner/work/milvus-backup/milvus-backup/core/paramtable/base_table.go:75 +0x5e
github.com/zilliztech/milvus-backup/cmd.glob..func1(0x1f5d480?, {0xf25595?, 0x2?, 0x2?})
        /home/runner/work/milvus-backup/milvus-backup/cmd/create.go:26 +0xc5
github.com/spf13/cobra.(*Command).execute(0x1f5d480, {0xc000522b60, 0x2, 0x2})
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:876 +0x67b
github.com/spf13/cobra.(*Command).ExecuteC(0x1f5e100)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:990 +0x3b4
github.com/spf13/cobra.(*Command).Execute(...)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:918
github.com/zilliztech/milvus-backup/cmd.Execute()
        /home/runner/work/milvus-backup/milvus-backup/cmd/root.go:24 +0x71
main.main()
        /home/runner/work/milvus-backup/milvus-backup/main.go:17 +0x17
```

Seems the config file should force to configs dir? https://github.com/zilliztech/milvus-backup/blob/v0.2.2/core/paramtable/base_table.go#L110


### Expected Behavior

_No response_

### Steps To Reproduce

```bash
# Use absolute config path, which should not in configs directory.


./milvus-backup --config $(pwd)/backup.yaml create -n backup-2023-04-26
```


### Environment

_No response_

### Anything else?

_No response_

--- a/core/paramtable/base_table.go
+++ b/core/paramtable/base_table.go
@@ -109,6 +109,10 @@ func (gp *BaseTable) initConfPath() string {
 	// check if user set conf dir through env
 	configDir, find := syscall.Getenv("MILVUSCONF")
 	if !find {
+		if _, err := os.Stat(defaultYaml); err == nil {
+			return path.Dir(defaultYaml)
+		}
+
 		runPath, err := os.Getwd()
 		if err != nil {
 			panic(err)
@@ -174,7 +178,7 @@ func (gp *BaseTable) LoadRange(key, endKey string, limit int) ([]string, []strin
 
 func (gp *BaseTable) LoadYaml(fileName string) error {
 	config := viper.New()
-	configFile := gp.configDir + fileName
+	configFile := fmt.Sprintf("%s/%s", strings.TrimRight(gp.configDir, "/"), path.Base(fileName))
 	if _, err := os.Stat(configFile); err != nil {
 		panic("cannot access config file: " + configFile)
 	}

